docs/linting/README.md

```diff
@@ -0,0 +1,99 @@
+# Linting
+As part of the build process we run a few linters to catch common programming errors. Linters are run only when working with the package (directory) structure. All linters are run via the following script:
```

Comment:
Add installation guide to install flake8, mypy and docker . Those are required in order to run `pkg_dev_test_tasks.py`